clients/mobile/src/database/Migration.ts

```diff
@@ -0,0 +1,55 @@
+import { DB, ItMigration, Schema } from 'react-native-sqlite-manager'
+
+export default class Migration extends ItMigration {
```

Comment:
@antoni0dev why there is `Migration` class here and create the schema in code?  I was thinking the migration should pick up the changes in `1.ts` file instead , right?